examples/stable/kafka/kafka-blueprint.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: kafka-connect-blueprint
+  namespace: kasten-io
+actions:
+  backup:
+    type: Deployment
+    phases:
+    - func: KubeTask
+      name: setupPhase
+      args:
+        namespace: kafka
+        podOverride:
+          containers:
+          - name: container
+            imagePullPolicy: IfNotPresent
+            livenessProbe:
+              httpGet:
+                path: /connectors
+                port: 8083
+              initialDelaySeconds: 60
+              periodSeconds: 5
+        image: akankshakumari393/confluent-kafka-s3sink:latest
+        command:
+          - bash
+          - -o
+          - errexit
+          - -o
+          - pipefail
+          - -c
+          - |
+            mkdir /tmp/config
+            {{- if .Profile.Credential.KeyPair }}
+            export AWS_SECRET_KEY="{{ .Profile.Credential.KeyPair.Secret }}"
+            export AWS_ACCESS_KEY="{{ .Profile.Credential.KeyPair.ID }}"
+            {{- else }}
+            export AWS_SECRET_KEY="{{ .Profile.Credential.Secret.Data.aws_secret_access_key | toString }}"
+            export AWS_ACCESS_KEY="{{ .Profile.Credential.Secret.Data.aws_access_key_id | toString }}"
+            {{- end }}
+            region="{{ .Profile.Location.Region }}"
+            bucket="{{ .Profile.Location.Bucket }}"
+            s3config="{{ index .Object.data "s3ConnectorConfiguration.properties" | toString }}"
+            echo "$s3config" > /tmp/config/s3config.properties
+            echo "s3.region= $region" >> /tmp/config/s3config.properties
+            echo "s3.bucket.name= $bucket" >> /tmp/config/s3config.properties
+            kafkaConfig="{{ index .Object.data "kafkaConfiguration.properties" | toString }}"
+            echo "$kafkaConfig" > /tmp/config/kafkaConfig.properties
+            sh /bin/connect-standalone /tmp/config/kafkaConfig.properties /tmp/config/s3config.properties
```

Comment:
we have file in the configmap right, if yes. cant we just pass s3config directly here to `connect-standalone`. instead of reading separate entries from configmap and then creating another config file.

Reply:
configuration file need to hold information about region and bucket which we get from Profile. So after Reading the file Region and bucket variables are appended to it and later on passed to `connect-standalone`